libs/application/template-api-modules/src/lib/environments/environment.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  defaultclosedDate: new Date(
+    process.env.PARTY_ENDORSEMENTLISTS_DEFAULT_CLOSED_DATE ||
+      '2021-09-15T00:00:00.000Z',
+  ),
+}
```

Comment:
Not sure if we want an environments file here for template specific configuration. Think we should move this to the template api folder for now.